src/components/railBottom/zoomControl/ZoomControl.vue

```diff
@@ -18,9 +18,7 @@
       data-cy="zoom-reset-control"
       v-b-tooltip.hover
       :title="$t('Reset to initial scale')"
-    >
-      {{ percentageText }}
-    </button>
+    >{{ percentageText }}</button>
```

Comment:
We were comparing ` 100% ` with the spaces included. Fixing the indent solved the issue